Assets/MRTK/Core/Interfaces/CameraSystem/IMixedRealityCameraSystem.cs

```diff
@@ -19,5 +19,12 @@ public interface IMixedRealityCameraSystem : IMixedRealityEventSystem, IMixedRea
         /// Is the current camera displaying on an opaque (VR / immersive) or a transparent (AR) device
         /// </summary>
         bool IsOpaque { get; }
+
+        /// <summary>
+        /// Override the camera's projection matrices for a smaller field of view
+        /// but rendered content will have more detail.  If holograms are not stable,
+        /// change the Stereo Rendering Mode from "Single Pass Instanced" to "Multi Pass."
+        /// </summary>
+        bool ProjectionOverrideEnabled { get; set; }
```

Comment:
Is the term Reading Mode well known enough that maybe we want to name it that (instead of specifically how this is implemented under the covers i.e. overriding projection matrices?) 

https://docs.microsoft.com/en-us/hololens/hololens2-display (ctrl + F reading mode).

I could swing either way, but I figured it would be worth agreeing on here (I think that ProjectOverrideEnabled may not really be descriptive of what it's actually giving you, even though it is descriptive of what it's doing).

Reply:
I'd been flip-flopping on this, mainly since it has uses beyond reading text.  But since it's in the docs I'm happy to go with ReadingMode for consistency.

Reply:
Renamed everything to ReadingMode or some variation